dendrite-sample.yaml

```diff
@@ -273,6 +273,9 @@ media_api:
       height: 480
       method: scale
 
+# Configuration for the Relay API.
```

Comment:
I don't think we should add this, the sample is more for usage with a global connection pool.
May be worth to add a sample file for SQLite instead, where we definitely need this section including the `database.connection_string`

Reply:
I see. Happy to remove this change here if you intend to create a SQLite sample file in another PR.